core/src/test/scala/de/frosner/dds/core/DDSTest.scala

```diff
@@ -34,6 +37,7 @@ class DDSTest extends FlatSpec with Matchers with MockFactory with BeforeAndAfte
     new SparkContext(conf)
   }
   private val sql: SQLContext = new SQLContext(sc)
+  import sql.implicits._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/frank_7/spawncamping-dds/file/2315269598/issues/source?bid=3046234&fileBranchId=3202429#l40)